src/services/payment-flow/index.ts

```diff
@@ -8,6 +8,7 @@ import {
 import { InvalidLightningPaymentFlowStateError, PaymentFlow } from "@domain/payments"
 import { paymentAmountFromNumber, WalletCurrency } from "@domain/shared"
 import { safeBigInt } from "@domain/shared/safe"
+import { parseRepositoryError } from "@services/mongoose/utils"
```

Comment:
this file (ie `src/services/payment-flow`) should be under `mongoose` like all the other repositories.

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/1707/commits/a10d5f494d8e8f4ba12e260937ce5e9e261a03ca